storage/mysql/tree_storage.go

```diff
@@ -252,6 +264,16 @@ func (t *treeTX) getSubtrees(ctx context.Context, treeRevision int64, nodeIDs []
 }
 
 func (t *treeTX) storeSubtrees(ctx context.Context, subtrees []*storagepb.SubtreeProto) error {
+	if glog.V(4) {
+		glog.Infof("storeSubtrees(")
+		for _, s := range subtrees {
+			glog.V(4).Infof("  prefix: %x, depth: %d", s.Prefix, s.Depth)
```

Comment:
Elsewhere you've removed the `V(x)` bit if you're inside an `if` block which already checked that